### PR TITLE
MatchData Admin Maintenance

### PR DESCRIPTION
**Pull request by KyleD0711**
Finally finished the matchdata stuff

### CODE REVIEW

File: app/controllers/metric.controller.js [hunk 1]
```diff
@@ -156,6 +156,19 @@ exports.getAllForTitle = async (req, res) => {
     });
 };
 
+exports.findAllMetricsForTitle = async (req, res) => {
+  await metricUtils
```
Comment (JonathanBooker912) [inline app/controllers/metric.controller.js:160]: Can you add an optional query parameter here to filter metrics by type? I.e. get only match metrics or get only player metrics

**Review by JonathanBooker912 — CHANGES_REQUESTED** [review]
Plz

File: app/controllers/metric.controller.js (hunk 1 above)
Comment (KyleD0711) [reply app/controllers/metric.controller.js:160]: Completed

**Review by JonathanBooker912 — APPROVED** [review]
ehe

**Pull request merged**